diagnostic_rt_qpcr/libraries/preparert_qpcrdefaults/source.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+module PrepareRT_qPCRDefaults
+
+  OUTPUT_PLATE = 'PCR Plate'
+  TEMPLATE = 'Template'
+  OPTIONS = 'options'
+  RESP_SAMPLE = 'Respiratory Sample'
+  METHOD = 'Method'
+  INPUT_PLATE = 'Input PCR Plate'
+
+  # Default parameters that are applied equally to all operations.
+  #   Can be overridden by:
+  #   * Associating a JSON-formatted list of key, value pairs to the `Plan`.
+  #   * Adding a JSON-formatted list of key, value pairs to an `Operation`
+  #     input of type JSON and named `Options`.
+  def default_job_params
+    {
+      negative_template_location: 'NTC',
+      negative_template_location: [0, 0],
+      positive_template_control: 'nCoVPC',
+      positive_template_location: [0, 11]
+    }
+  end
+
+
+  def default_operation_params
```

Comment:
This object is too complex for operation parameters. Pass a single key-value that is used to retrieve one of these objects. See https://github.com/dvnstrcklnd/aq-pcr-models/blob/0e367a4b3727ef01e29f541580e437bc1fc1c6b5/libraries/PCRCompositionDefinitions.rb#L246